vmdk_plugin/plugin.go

```diff
@@ -208,11 +209,50 @@ func (d *vmdkDriver) unmountVolume(name string) error {
 // (until Mount is called).
 // Name and driver specific options passed through to the ESX host
 func (d *vmdkDriver) Create(r volume.Request) volume.Response {
-	err := d.ops.Create(r.Name, r.Options)
+	// Defaults to ext4 if not specified
+	if _, ok := r.Options["fstype"]; ok == false {
```

Comment:
nit: would be better to have a global const with "fstype" string and "ext4" string, just to have it all in one place